src/hooks/test/use-arrow-key-navigation-test.js

```diff
@@ -1,302 +0,0 @@
-import { options as preactOptions, render } from 'preact';
-import { useRef } from 'preact/hooks';
-import { act } from 'preact/test-utils';
-
-import { useArrowKeyNavigation } from '../use-arrow-key-navigation';
-import { waitFor } from '../../test-util/wait';
-
-function Toolbar({ navigationOptions = {} }) {
-  const containerRef = useRef();
-
-  useArrowKeyNavigation(containerRef, navigationOptions);
-
-  return (
-    <div ref={containerRef} data-testid="toolbar">
-      <button data-testid="bold">Bold</button>
-      <button data-testid="italic">Italic</button>
-      <button data-testid="underline">Underline</button>
-      <a href="/help" target="_blank" data-testid="help">
-        Help
-      </a>
-    </div>
-  );
-}
-
-describe('useArrowKeyNavigation', () => {
-  let container;
-
-  beforeEach(() => {
-    container = document.createElement('div');
-    document.body.append(container);
-    renderToolbar();
-  });
-
-  afterEach(() => {
-    container.remove();
-  });
-
-  // Workaround for an issue with `useEffect` throwing exceptions during
-  // `act` callbacks. Can be removed when https://github.com/preactjs/preact/pull/3530 is shipped.
-  let prevDebounceRendering;
-  beforeEach(() => {
-    prevDebounceRendering = preactOptions.debounceRendering;
-  });
-  afterEach(() => {
-    preactOptions.debounceRendering = prevDebounceRendering;
-  });
-
```

Comment:
I double-checked that this has shipped and, yep, it has. I removed this from the updated tests.